translations/messages.en.yaml

```diff
@@ -4,6 +4,15 @@ Email: Email
 Password: Contraseña
 Login: Iniciar sesión
 
+navbar.home: Home
+navbar.opportunity: Opportunities
+navbar.agents: Agents
+navbar.events: Events
+navbar.spaces: Spaces
+navbar.projects: Projects
+navbar.notifications: Notifications
+navbar.login: Login
```

Comment:
Pronto, é exatamente isso só que sem precisar do `navbar.`, pois é aquilo que falei mais cedo na daily: "quando o texto a ser traduzido for apenas uma unica palavra não precisa usar o chaveamento"